R/deployApp.R

```diff
@@ -157,6 +155,25 @@ deployApp <- function(appDir = getwd(),
   check_directory(appDir)
   check_string(appName, allow_null = TRUE)
 
+  if (!is.null(appPrimaryDoc)) {
+    check_string(appPrimaryDoc)
+    if (!file.exists(file.path(appDir, appPrimaryDoc))) {
+      cli::cli_abort("`appPrimaryDoc` not found inside `appDir`")
+    }
+  }
+
+  if (!is.null(appSourceDoc)) {
+    lifecycle::deprecate_warn(
+      when = "0.9.0",
```

Comment:
Note that I'm assuming the next release will be 0.9.0; I think this is a reasonable assumption as it's likely to have some bigger new features.